kube/pod_test.go

```diff
@@ -1,19 +1,25 @@
 package kube
 
 import (
+	"bytes"
 	"encoding/base64"
 	"os"
 	"path/filepath"
+	"strings"
 	"testing"
 
+	yaml "gopkg.in/yaml.v2"
```

Comment:
This should probably go live with `assert` and other packages at the end of the `import` section